main/views.py

```diff
@@ -1,6 +1,11 @@
 from django.shortcuts import render
+from django.contrib.auth import logout
 
 
-# Create your views here.
 def homePage(request):
     return render(request, 'main/homePage.html')
+
+
+def sign_out(request):
+    logout(request)
+    return render(request, 'main/homePage.html')
```

Comment:
```suggestion
from django.shortcuts import redirect
from django.contrib.auth.decorators import login_required

@login_required
def sign_out(request):
    logout(request)
    return redirect("home_page")
```

I think it may be better to use Django builtin  [`LogoutView` ](https://docs.djangoproject.com/en/4.2/topics/auth/default/#all-authentication-views).

If you wanna keep the manual implementation, I suggest adding [`@login_required decorator`](https://docs.djangoproject.com/en/4.2/topics/auth/default/#the-login-required-decorator), and also instead of manually using render function and typing URL manually maybe instead we can redirect to homepage